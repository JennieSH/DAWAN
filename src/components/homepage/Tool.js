import React from "react";


class Tool extends React.Component{
    render(){
        const text = [
            {
                icon : "record_voice_over",
                title : "Text To Speech",
                inner : "We provide standard pronunciations in any language"
            },
            {
                icon : "mic",
                title : "Record",
                inner : "It allows you to record your voice and compare with Google pronunciation quickly."
            },
            {
                icon : "edit",
                title : "Writing Board",
                inner : "Writing Board is the best way to learning how to write the correct character."
            },
        ]
        return(
            text.map((tool, index)=>                       
                <div className="tool card" key={index}>
                    <div className="card-image" >             
                        <i className="material-icons">{tool.icon}</i> 
                    </div>
                    <div className="card-content">
                        <span className="card-title center ">{tool.title}</span>
                        <p>{tool.inner}</p>
                    </div>                      
                </div>    
            )
        )         
    }
}
export default Tool