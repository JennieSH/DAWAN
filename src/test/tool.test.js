import React from "react";
import renderer from "react-test-renderer";
import Tool from "../components/homepage/tool";

test("Snapshot for authBtn Component", () => {
  const guest = renderer.create(<Tool/>).toJSON();
  expect(guest).toMatchSnapshot();
});