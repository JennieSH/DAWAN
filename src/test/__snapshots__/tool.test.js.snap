// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for authBtn Component 1`] = `
Array [
  <div
    className="tool card"
  >
    <div
      className="card-image"
    >
      <i
        className="material-icons"
      >
        record_voice_over
      </i>
    </div>
    <div
      className="card-content"
    >
      <span
        className="card-title center "
      >
        Text To Speech
      </span>
      <p>
        We provide standard pronunciations in any language
      </p>
    </div>
  </div>,
  <div
    className="tool card"
  >
    <div
      className="card-image"
    >
      <i
        className="material-icons"
      >
        mic
      </i>
    </div>
    <div
      className="card-content"
    >
      <span
        className="card-title center "
      >
        Record
      </span>
      <p>
        It allows you to record your voice and compare with Google pronunciation quickly.
      </p>
    </div>
  </div>,
  <div
    className="tool card"
  >
    <div
      className="card-image"
    >
      <i
        className="material-icons"
      >
        edit
      </i>
    </div>
    <div
      className="card-content"
    >
      <span
        className="card-title center "
      >
        Writing Board
      </span>
      <p>
        Writing Board is the best way to learning how to write the correct character.
      </p>
    </div>
  </div>,
]
`;
